<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetlistBrowserConfigPage2</class>
 <widget class="QFrame" name="NetlistBrowserConfigPage2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netlist Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Marker Appearance</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nets, circuits and devices are highlighted using the given styles.
Nets can be highlighted using the drawing colors for original layers with a higher or lower
intensity, drawn with the highlight color or auto-colored with a cycling color from the palette.
Non-net objects are drawn using the highlight color.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>556</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="3" column="0" colspan="4">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Net auto-coloring</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>16</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="lay::ColorButton" name="color_pb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>The color in which the rulers are drawn</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>or</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="brightness_cb">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="brightness_sb">
              <property name="minimum">
               <number>-100</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="brightness_label">
              <property name="text">
               <string>% intensity increase (for net shapes on original layers)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>201</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Highlight color</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>16</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="cycle_colors_cb">
              <property name="text">
               <string>Auto-color with palette</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc0">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc1">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc5">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc6">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc3">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc7">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc4">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cc2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="2" column="3">
          <widget class="QLabel" name="vs_lbl">
           <property name="text">
            <string>Vertex size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QLabel" name="textLabel1_4">
           <property name="text">
            <string>Line width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>pixel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6" rowspan="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>71</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="vs_le">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2">
          <widget class="QLineEdit" name="lw_le">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="7">
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>590</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="lay::DitherPatternSelectionButton" name="stipple_pb">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Stipple </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="halo_cb">
           <property name="text">
            <string/>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Halo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" rowspan="2">
          <widget class="QLabel" name="textLabel2_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>pixel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>lay::DitherPatternSelectionButton</class>
   <extends>QPushButton</extends>
   <header>layWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>lay::ColorButton</class>
   <extends>QPushButton</extends>
   <header>layWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stipple_pb</tabstop>
  <tabstop>lw_le</tabstop>
  <tabstop>halo_cb</tabstop>
  <tabstop>vs_le</tabstop>
  <tabstop>color_pb</tabstop>
  <tabstop>brightness_cb</tabstop>
  <tabstop>brightness_sb</tabstop>
  <tabstop>cycle_colors_cb</tabstop>
  <tabstop>cc0</tabstop>
  <tabstop>cc1</tabstop>
  <tabstop>cc5</tabstop>
  <tabstop>cc6</tabstop>
  <tabstop>cc3</tabstop>
  <tabstop>cc7</tabstop>
  <tabstop>cc4</tabstop>
  <tabstop>cc2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>brightness_cb</sender>
   <signal>clicked(bool)</signal>
   <receiver>brightness_sb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightness_cb</sender>
   <signal>clicked(bool)</signal>
   <receiver>brightness_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
